src/connectors/clear-refinements/connectClearRefinements.ts

```diff
@@ -154,6 +157,33 @@ const connectClearRefinements: ClearRefinementsConnector = function connectClear
       dispose() {
         unmountFn();
       },
+
+      getWidgetRenderState(renderState, { scopedResults }) {
+        const attributesToClear = scopedResults.reduce<
```

Comment:
this gets calculated both here and in `refine` itself still. Should we call it in the beginning of `refine` and use its value, instead of computing it again instead?

Reply:
Addressed in bfbe04dcf11a0b3af882d30830bb933b7d9a82fc.